apps/aecore/src/aec_blocks.erl

```diff
@@ -144,9 +144,9 @@ serialize_for_network(B = #block{}) ->
 
 serialize_to_map(B = #block{}) ->
     #{<<"height">> => height(B),
-      <<"prev_hash">> => base64:encode(prev_hash(B)),
-      <<"state_hash">> => base64:encode(B#block.root_hash),
-      <<"txs_hash">> => base64:encode(B#block.txs_hash),
+      <<"prev_hash">> => aec_base58c:encode(block_hash, prev_hash(B)),
+      <<"state_hash">> => aec_base58c:encode(block_hash, B#block.root_hash),
```

Comment:
Distinct type e.g. block_state_hash? This is hash of the state (i.e. summary of all state trees in a block), not hash of block header.